### ISSUE TITLE
[Bug] - Can't see project name clearly in the project switch list

### ISSUE THREAD

**Issue by trofoto** [dinbtechit/vscode-theme#159]
## Describe the bug

Can't see the selected project name clearly Under the VSCode Light Modern theme when enabled New UI. 
Because the color of project name is similar with the background color.
It doesn't occur at the top of menus (above _Open Projects_). 

## Platform & Language Details 
- Language: Java
- IDE Details:  
```
IntelliJ IDEA 2023.2.2 (Ultimate Edition)
Build #IU-232.9921.47, built on September 13, 2023
```

## Jetbrains Screenshot

Color with problem, the background color is light blue but color of name is white
![image](https://github.com/dinbtechit/vscode-theme/assets/8140291/efc57318-0cf9-4db7-92eb-98b8be828cab)

Normal background color, we could see the original background color is some kind of dark blue and color of name is also white
![image](https://github.com/dinbtechit/vscode-theme/assets/8140291/a4c810fd-f3d7-4ff6-8c08-c14861ab1ff6)

## Additional context/Logs
I think it's ok if we follow the default background color

**Comment by dinbtechit** [comment]
Thank you for reporting this issue. Hopefully we can fix this in the next release. PRs are welcome tho if you have time. Thanks!

**Comment by dinbtechit** [comment]
Fixed in v1.10.7
